### DISCUSSION TITLE
Help to add an ads.txt file

### DISCUSSION THREAD

**Discussion by bpwhite** [requarks/wiki#5418] [Help / Questions] [answered]
Hi,

Google is requesting that I implement an ads.txt file, which now looks like it is standard for AdSense. But I am not sure how to create a static page on my wiki.js installation. Any help to finding the right place to upload an ads.txt file (it has to be in the document root, e.g., www.google.com/ads.txt in order to work)

Thanks.

**Comment by NGPixel** [comment]
Don't create a page. Instead create a ads.txt file on your desktop and upload it as an asset to the root.

**Reply by bpwhite** [reply]
I did upload to root directory of the assets folder but it still asks to create a page. Whereas the favicon.ico is there and that one works. So it must be something with the .txt that is making it not work?

**Comment by NGPixel** [comment, marked as answer]
Update to 2.5.285. I added a field in Admin > General to configure the page extensions. Simply remove "txt" from the list and your file will be accessible correctly.

**Reply by bpwhite** [reply]
Thanks looks like this worked.
